qemu/qmp.go

```diff
@@ -242,8 +245,19 @@ type MigrationStatus struct {
 	XbzrleCache  MigrationXbzrleCache     `json:"xbzrle-cache,omitempty"`
 }
 
+// SchemaInfo represents all QMP wire ABI
+type SchemaInfo struct {
+	MetaType string `json:"meta-type"`
+	Name     string `json:"name"`
+}
+
 func (q *QMP) readLoop(fromVMCh chan<- []byte) {
 	scanner := bufio.NewScanner(q.conn)
+	if q.cfg.MaxCapacity != 0 {
```

Comment:
Just wondering why this change is needed?  Is it because lines can be received via QMP that are greater than 64kb?

Reply:
Yes, QMP command `query-qmp-schema` will return 120kb almost.